examples/hqca_circuit.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 from typing import Any, Dict, List
 
-from opensquirrel.squirrel_ir import Qubit
+from opensquirrel.ir import Qubit
```

Comment:
please check whether this has indeed been updated to `ir`. I asked roberto to revert everything.

Reply:
Reverted it

Reply:
Seems like all other changes besides the rename back to opensquirrel were left untouched

Reply:
if that's the case, can you please also change the calls in the other examples?

Reply:
What calls exactly? I've changed the import statements to match `ir` but don't see any other place where a changes are required.

Reply:
never mind, I thought there were additional example files that had calls to the opensquirrel IR, but there aren't.